qir/qsharp/MeasurementSeries/Makefile

```diff
@@ -1,7 +1,7 @@
 all: ungrouped
 
 ungrouped: QSharpVersion/qir/Example.ll
-	${QAT_BINARY} --apply -S -O3 --always-inline --unroll-loops --disable-grouping --profile default QSharpVersion/qir/Example.ll  > ungrouped.ll
+	${QAT_BINARY} --apply -S --always-inline --unroll-loops --disable-grouping --adaptor default QSharpVersion/qir/Example.ll  > ungrouped.ll
```

Comment:
FYI: 
In other similar fragments the line ending is `|| echo "failed!"`. 
But here the ending is `> ungrouped.ll`. 
That looks suspicious.

Reply:
This is recorded in #140